jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/component/DataGridLoader.java

```diff
@@ -36,4 +37,20 @@ protected void setupDataProvider(GridDataHolder holder) {
             resultComponent.setItems(new EmptyDataGridItems(holder.getMetaClass()));
         }
     }
+
+    @Override
+    protected void loadActions() {
+        loadActionsAttributes();
+        super.loadActions();
+    }
+
+    protected void loadActionsAttributes() {
```

Comment:
the same code as in `TreeDataGridLoader`, why not adding to base class?

Reply:
Because it uses `DataGrid.getActionsSupport()`/`TreeDataGrid.getActionsSupport()` to set the attribute and I didn't want to move `getActionsSupport()` method to `EnhancedDataGrid` interface.